### ISSUE TITLE
Feature Request: I want a separate session for each instance of index.php on my server.

### ISSUE THREAD

**Issue by Habimm** [jcampbell1/simple-file-manager#74]
I have the following problem: On my server, I have multiple instances of index.php each of which have different passwords. But when I log in into one of them, the server also lets me log in into all the others although I might not know their passwords. This is a security bug if I use multiple instances of index.php.

**Comment by dura4ok** [comment]
Hi. I added a new variable $projectname in which I save the directory name. And changed the code section where the session is being created.
```php
 if(!$_SESSION['_sfm_allowed'.$PROJECT_NAME]) {
                // sha1, and random bytes to thwart timing attacks.  Not meant as secure hashing.
                $t = bin2hex(openssl_random_pseudo_bytes(10));
                if($_POST['p'] && sha1($t.$_POST['p']) === sha1($t.$PASSWORD)) {
                        $_SESSION['_sfm_allowed'.$PROJECT_NAME] = true;
                        header('Location: ?');
                }
                echo '<html><body><form action=? method=post>PASSWORD:<input type=password name=p /></form></body></html>';
                exit;
        }
```

**Comment by Habimm** [comment]
Cool! That's also similar how I solved it myself in #75
